Content.Server/Radio/EntitySystems/GetInsteadIdCardNameSystem.cs

```diff
@@ -0,0 +1,62 @@
+using Content.Server.Radio.EntitySystems;
+using Content.Shared.Access.Components;
+using Content.Shared.Inventory;
+using Content.Shared.PDA;
+using Content.Shared.Silicons.Borgs.Components;
+using System.Globalization;
+
+public sealed class GetInsteadIdCardName : EntitySystem
+{
+    [Dependency] private readonly InventorySystem _inventorySystem = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<BorgChassisComponent, GetInsteadIdCardNameEvent>(OnGetBorgName);
+        SubscribeLocalEvent<InventoryComponent, GetInsteadIdCardNameEvent>(OnGetIdCardName);
+    }
+
+    private void OnGetBorgName(EntityUid uid, BorgChassisComponent component, ref GetInsteadIdCardNameEvent args)
```

Comment:
И перенести в `BorgSystem`
```suggestion
    private void OnGetBorgName(Entity<BorgChassisComponent> ent, ref GetInsteadIdCardNameEvent args)
```